crates/bevy_render/src/render_resource/pipeline_cache.rs

```diff
@@ -624,16 +659,41 @@ impl PipelineCache {
     pub fn queue_compute_pipeline(
         &self,
         descriptor: ComputePipelineDescriptor,
+    ) -> CachedComputePipelineId {
+        // SAFETY: We are setting `unchecked` to `false`, that's always safe
+        unsafe { self.queue_compute_pipeline_ex(descriptor, false) }
+    }
+
+    /// Like [`Self::queue_compute_pipeline()`], but disables runtime checks for the shader.
+    ///
+    /// # Safety
+    ///
```

Comment:
This should fully detail the safety invariant by itself instead of linking to wgpu's docs.